src/css/styles.scss

```diff
@@ -165,32 +145,24 @@ h4 {
 }
 
 #steps-card-container {
-  background-color: #F5634B;
-  grid-area: steps;
-  margin: 0.5vw;
+  @include card-container(#F5634B, steps);
```

Comment:
I am happy to take on creating semantically named variables for each of the hex codes.